### PR TITLE
7335: Cleanup code, filenames, and documentation

### PR DESCRIPTION
**Pull request by dsblank**
First part of cleanup, changes:

MediaObject -> Media
mediaobj -> media
mediaobject -> media

### CODE REVIEW

File: gramps/gui/clipboard.py [hunk 1]
```diff
@@ -1644,7 +1644,7 @@ def edit_obj(self, objclass, handle):
                              self.uistate, [], ref)
                 except WindowActiveError:
                     pass
-        elif objclass in ['Media', 'MediaObject']:
+        elif objclass in ['Media', 'Media']:
```
Comment (Nick-Hall) [inline gramps/gui/clipboard.py]: The list is no longer required.  There are 3 other similar lines.

**Comment by Nick-Hall** [comment]
Thanks.  This is a nice tidy up.

File: gramps/gui/clipboard.py (hunk 1 above)
Comment (dsblank) [inline gramps/gui/clipboard.py]: Good catch! I thought I had caught all of those. Will fix and merge.

**Comment by Nick-Hall** [comment]
Why commit a merge in the middle?  It makes a mess of the history.

**Comment by dsblank** [comment]
Oh, I should have done a "git rebase"?

**Comment by Nick-Hall** [comment]
Yes.

**Comment by dsblank** [comment]
Ok, thanks. I didn't think that would show in a PR.

**Pull request merged**